README.adoc

```diff
@@ -117,20 +117,41 @@ method List {
 	// Order criteria.
 	in Order String
 
+        // Kind of the result - "ClusterList".
+        out Kind String
+
 	// Total number of items of the collection that match the search criteria,
 	// regardless of the size of the page.
-	in Total Integer
+	out Total Integer
 
 	// Retrieved list of clusters.
 	out Items []Cluster
 }
 ----
 
 Methods have _parameters_ defined by their direction (_in_ or _out_), their
-name and their type. In the above example there are five input parameters
-(named `Page`, `Size`, `Search` and `Order`) and one output parameter (named
+name and their type. In the above example there are four input parameters
+(named `Page`, `Size`, `Search` and `Order`) and four output parameter (named
```

Comment:
Should be: `two output parameters (named `Total` and `Items`).

Reply:
oh, I forgot to edit this part.  I meant four, including the in-out Page and Size.